CSharp/Library/Microsoft.Bot.Builder/ConnectorEx/BotData.cs

```diff
@@ -65,7 +63,7 @@ public interface IBotDataStore<T>
         /// <param name="botStoreType"> The bot store type.</param>
         /// <param name="cancellationToken"> The cancellation token.</param>
         /// <returns>Bot record that is stored for this key, or "empty" bot record ready to be stored</returns>
-        Task<T> LoadAsync(IAddress key, BotStoreType botStoreType, CancellationToken cancellationToken);
+        Task<T> LoadAsync(IAddress key, BotStoreType botStoreType, CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
Better change might be to removing default arguments from other interfaces rather than adding default(CancellationToken) to all interfaces. 

---
In reply to: [113984240](https://github.com/Microsoft/BotBuilder/pull/2457#discussion_r113984240) [](ancestors = 113984240)